uni/lib/model/entities/library_occupation.dart

```diff
@@ -0,0 +1,47 @@
+/// Overall occupation of the library
+class LibraryOccupation {
+  int occupation = 0;
+  int capacity = 0;
+  List<FloorOccupation> floors = [];
+
+  LibraryOccupation(this.occupation, this.capacity);
```

Comment:
Set the default values in the constructor, and remove them from the field declaration.